pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "libyear"
+version = "0.1.0"
+description = "A python library to calculate libyear for python depenencies"
+authors = []
+readme = "README.md"
+keywords = [ "backmarket", "libyear",]
+
+
+[tool.poetry.dependencies]
+python = "^3.12"
+requests = "^2.31.0"
+prettytable = "^3.10.0"
+python-dateutil = "^2.9.0.post0"
+setuptools = "^69.2.0"
+packaging = "^24.0"
+coverage = "^7.0.5"
+
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.0.0"
+pytest-sugar = "^0.9.4"
+pytest-cov = "^4.0.0"
+pytest-asyncio = "^0.23.0"
+pytest-runner = "^6.0.1"
+
```

Comment:
We should add the requirements from `test_requirements.txt`  here and delete that file! (we are missing `pytest-vcr`).

Reply:
Without pytest-vcr I was able to run the test anyway 🤔 , but I'm gonna add it